doc/index.rst

```diff
@@ -74,10 +74,30 @@ and simply replace ``'default.qubit'`` with any of the available Qiskit devices,
 You can also try to run tutorials, such as the qubit rotation tutorial, on actual quantum hardware by
 using the ``'qiskit.ibmq'`` device.
```

Comment:
What do you think of the existing text here (and above)?

Part of me is wondering if we should remove it, and simply have the demo cards and a link to the demonstrations+filter?

Reply:
I tried adjusting the text, let me know what you think - I actually removed the link to the filter since we're going to showcase those same ones here anyways.

Reply:
Oh yeah looks way better!